### ISSUE TITLE
fix use of deprecated code

### ISSUE THREAD

**Issue by pjfanning** [apache/incubator-pekko-http#345]
```
[warn] /Users/pj.fanning/code/incubator-pekko-http/http-core/src/main/scala/org/apache/pekko/http/scaladsl/model/HttpEntity.scala:666:23: method getFirst in class Attributes is deprecated (since Akka 2.5.7): Attributes should always be most specific, use get[T]
[warn]           _attributes.getFirst[SizeLimit] match {
```

**Comment by jrudolph** [comment]
Yep, we tried that before and did not find an easy solution. In the best case, we would dig in and understand what the differences are and understand how to fix the problems in Pekko HTTP.

**Comment by c4mdax** [comment]
hi @pjfanning! I would like to take this up as my first contribution to Pekko

Since PR #346 showed that migrating to get breaks the test suite due to differing behavior, I'll go ahead and remove the @deprecated annotation from getFirst as you suggested

I'll open a PR shortly so you can review it. Let me know if you need anything else!
